lib/actions/transform.js

```diff
@@ -37,7 +56,9 @@ async function processAction(msg, cfg) {
 
     return Promise.resolve();
   }
+
   this.logger.info('1 message will be send to the next step');
+  this.logger.info(JSON.stringify(result));
```

Comment:
Can you add a comment to this log statement for context in the logs? Same for line 24 above